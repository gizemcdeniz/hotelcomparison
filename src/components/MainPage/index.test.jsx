// import React from "react";
// import renderer from "react-test-renderer";
// import { BrowserRouter as Router } from "react-router-dom";
// import GenderViolenceMain from "./index";

// it("renders correctly and matches the snapshot", () => {
//   const tree = renderer
//     .create(
//       <Router>
//         <GenderViolenceMain />
//       </Router>
//     )
//     .toJSON();
//   expect(tree).toMatchSnapshot();
// });
