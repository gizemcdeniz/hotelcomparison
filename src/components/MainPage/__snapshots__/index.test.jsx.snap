// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly and matches the snapshot 1`] = `
<div
  className="main-container container-md"
>
  <div
    className="row"
  >
    <div
      className="main-text col-md-6 col-12 order-md-first order-last"
    >
      <div>
        <h1>
          home.genderViolenceMain.notAloneStatement
        </h1>
        <h1
          dangerouslySetInnerHTML={
            Object {
              "__html": "home.genderViolenceMain.safePlaceStatement",
            }
          }
        />
      </div>
      <p>
        home.genderViolenceMain.subHeader
      </p>
      <a
        href="/seekhelp"
        onClick={[Function]}
      >
        <button
          type="button"
        >
          home.genderViolenceMain.button
        </button>
      </a>
    </div>
    <div
      className="col-md-6 col-12 order-md-last order-first"
    >
      <img
        alt="Stop Gender Violence"
        src="stop-gender-violence.svg"
      />
    </div>
  </div>
</div>
`;
